src/things.ts

```diff
@@ -112,6 +115,8 @@ async function getTasksFromThingsDb(
         ON TMTag.uuid = TMTaskTag.tags
     LEFT JOIN TMArea
         ON TMTask.area = TMArea.uuid
+    LEFT JOIN TMTask TMProject
```

Comment:
@olexs I was looking into extending your PR to also include headings. I cannot find the TMProject table in the database though. Apparently, projects are just special tasks and stored in the TMTask table, same thing with headings. However, your plugin seems to work, what am I missing?

Reply:
`TMProject` is not a separate table, but an alias for `TMTask`. The query self-joins `TMTask` to map tasks the their assigned projects. Actually, very good hint with headings - I need to add the same mechanic for those, currently they cause a bug that breaks the sync somewhat.